index.html

```diff
@@ -1521,6 +1479,55 @@ <h3>Hashlinks</h3>
 
     </section>
 
+    <section>
+      <h2>Zero-Knowledge Proofs</h2>
+      <em>This section is non-normative</em>
+        <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging digital proof formats. Each
+proof format has benefits and drawbacks. Many proof formats cannot reveal
+selected attribute values in a verifiable credential; they can only reveal all
```

Comment:
`selected attribute values in a`
-> `only selected attribute values from a`

(add `only`; change `in` to `from`)

Reply:
fixed in [5ff637c](https://github.com/w3c/vc-imp-guide/pull/32/commits/5ff637c784e5bec18463221499a55af27f8746e1)